### DISCUSSION TITLE
Can Miller remove records?

### DISCUSSION THREAD

**Discussion by YokojimaSkewers** [johnkerl/miller#1210] [Q&A] [answered]
Could I use Miller for removing records?

For example, I expect some command or something like

```
miller --csv --from a.csv remove '$A==c' then tee b.csv
```

with a.csv

```
A,B
a,b
c,d
e,f
```

outputs b.csv

```
A,B
a,b
e,f
```

**Comment by johnkerl** [comment, marked as answer]
@YokojimaSkewers I think you want `filter -x`:

```
$ mlr --csv --from a.csv filter -x '$A=="c"' then tee b.csv
A,B
a,b
e,f
```

See also https://miller.readthedocs.io/en/latest/reference-verbs/index.html#filter
